src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,207 @@
+package duke;
+
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.Scanner;
+
+/**
+ * <h1>duke.Duke!</h1>
```

Comment:
Maybe there's no need to mention the package name. One suggestion could be to package your code more.